swagger.yml

```diff
@@ -388,17 +380,19 @@ definitions:
         description: The phone number of the recipient in the following format +1<area code><10 digit phone number>
         pattern: '^[+]1\d{10}$'
         example: "+15145573849"
-      notes:
```

Comment:
swapped notes and email in the order it appears since it looks better when all the required fields are shown first